app/controllers/order_confirms_controller.rb

```diff
@@ -0,0 +1,30 @@
+class OrderConfirmsController < ApplicationController
+  before_action :load_order, :check_expiration
```

Comment:
User ko cần login vẫn có thể vào order ah em

Reply:
Dạ e cho khi nào thanh toán mới cần login

Reply:
ôi cha, order mà ko cần login, chú đùa ah
coi lại DB, tin tui thu tiền ko chú ơi